code/io/increaseHumanGEMVersion.m

```diff
@@ -42,23 +41,37 @@ function increaseHumanGEMVersion(bumpType)
 newVersion = num2str(newVersion,'%d.%d.%d');
 
 %Check if history has been updated:
-%fid     = fopen('../../history.md','r');
-%history = fscanf(fid,'%s');
-%fclose(fid);
-%if ~contains(history,['human' newVersion ':'])
-%    error('ERROR: update history.md first')
-%end
-%To be included
+fid     = fopen(fullfile(modelPath,'history.md'),'r');
+history = fscanf(fid,'%s');
+fclose(fid);
+if ~contains(history,['human' newVersion ':'])
+    error('ERROR: update history.md first')
+end
 
 %Load model:
-ymlFile=fullfile(modelPath,'model','Human-GEM.yml');
-ihuman = importYaml(ymlFile);
+ihuman = readYAMLmodel(fullfile(modelPath,'model','Human-GEM.yml'));
 
 %Include tag and save model:
 ihuman.version = newVersion;
 
-%Export model to multiple formats
-exportHumanGEM(ihuman,'Human-GEM',modelPath,{'mat', 'txt', 'xml', 'yml', 'xlsx'});
+%Check if it matches reactions.tsv, metabolites.tsv and genes.tsv
+fields = {'rxns','reactions';'mets','metabolites';'genes','genes'};
+for i=1:size(fields,1)
+    tsvList = importTsvFile(fullfile(modelPath,'model',[fields{i,2} '.tsv']));
+    Lia     = ismember(ihuman.(fields{i,1}), tsvList.(fields{i,1}));
+    dispEM(['The following ' fields{i,2} ' are in model.' fields{i,1} ...
```

Comment:
Here reports this error: those genes are deleted in the recent develop branch and also the genes.tsv

Error using dispEM
The following genes are in model/genes.tsv but not in model.genes:
	ENSG00000092529
	ENSG00000092964
	ENSG00000111237
	ENSG00000116783
	ENSG00000139977
	ENSG00000156269
	ENSG00000168137
	ENSG00000181652
	ENSG00000183665
	...and 5 more